### PR TITLE
Label Data Functions

### PR DESCRIPTION
**Pull request by ryan280**

### CODE REVIEW

**Comment by mpolimen** [comment]
data/index.js needs to be updated to reflect that label data is available now

File: src/data/labels.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+    },
+
+    /**
+     * Get a specific card from a board.
+     * @param {string} boardId The board's id.
+     * @param {string} cardId The card's id.
+     * @returns A card object
+     */
+    async getCardById(boardId, cardId){
```
Comment (mpolimen) [inline src/data/labels.js:165]: why does this exist in label data?

File: src/data/labels.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+        if(!boardId || !error_handler.checkObjectId(boardId)){
+            throw new Error('boardId is not valid.');
+        }
+        if(!cardId || !error_handler.checkObjectId(cardId)){
+            throw new Error('cardId is not valid.');
+        }
+        if(!text || !error_handler.checkNonEmptyString(text)){
+            throw new Error('Text must not be empty.');
+        }
+        if(!color || !error_handler.checkColor(color)){
```
Comment (mpolimen) [inline src/data/labels.js:210]: color may have to be randomly selected from pre-approved values when we actually implement

File: src/data/labels.js (hunk 1 above)
Comment (ryan280) [reply src/data/labels.js:165]: Its used in my other functions, Ill remove this from the exported modules

File: src/data/labels.js (hunk 2 above)
Comment (ryan280) [reply src/data/labels.js:210]: I can add this feature if we all want it, but I was thinking they would select the color when making the label.

**Comment by roccopolimen** [comment]
Merge for now, test will be soon

**Pull request merged**